### PR TITLE
Update history view

### PR DESCRIPTION
**Pull request by TW80000**
Update the history view to use `Element.select()` so that it updates when the underlying DAO changes.

### CODE REVIEW

File: src/foam/u2/history/HistoryView.js [hunk 1]
```diff
@@ -8,7 +8,10 @@ foam.CLASS({
   package: 'foam.u2.history',
   name: 'HistoryView',
   extends: 'foam.u2.View',
-  requires: [ 'foam.u2.history.HistoryItemView' ],
+  requires: [
+    'foam.u2.history.HistoryItemView',
```
Comment (kgrgreer) [inline src/foam/u2/history/HistoryView.js]: sort requires

File: src/foam/u2/Element.js [hunk 2]
```diff
@@ -1546,6 +1546,17 @@ foam.CLASS({
       return slot;
     },
 
+    /**
```
Comment (kgrgreer) [inline src/foam/u2/Element.js:1549]: Great!

File: src/foam/u2/history/HistoryView.js [hunk 3]
```diff
@@ -68,24 +73,15 @@ foam.CLASS({
...
+          return this.E().start('div')
+            .addClass('timelineRecord')
+            .start('div').addClass('timeline').end()
+            .call(function() {
+              view.historyItemView.outputRecord(this, record)
             })
-          })
-        });
+          .end();
+        }, false);
```
Comment (kgrgreer) [inline src/foam/u2/history/HistoryView.js]: Can remove: ", false"

**Comment by TW80000** [comment]
I signed it!

**Pull request merged**